inst/qml/GeneralizedLinearModel.qml

```diff
@@ -61,24 +61,24 @@ Form {
 		Group
 		{
 			title: qsTr("Model Fit")
-			CheckBox { name: "gofDeviance";	label: qsTr("Deviance goodness-of-fit test")}
-			CheckBox { name: "gofPearson";	label: qsTr("Pearson goodness-of-fit test")	}
+			CheckBox { name: "devianceGoodnessOfFit";	label: qsTr("Deviance goodness-of-fit test")}
+			CheckBox { name: "pearsonGoodnessOfFit";	label: qsTr("Pearson goodness-of-fit test")	}
 		}
 
 		Group
 		{
 			title: qsTr("Parameter Estimates")
 			CheckBox
 			{
-				name:		"coefEstimates"
+				name:		"coefficientEstimate"
```

Comment:
thinking about correspondence with the group label, how about `parameterEstimatesTable`?

Reply:
1. You do not need to, the "Table" is actually not needed here as well, the main point was that if the label is saying "parameters", then we should stick to that in the option name as well. (or alternatively change the label)
2. Yes, we should think about it. At the very least, it would be good that linear regression, logistic regression, and glms use the same labels + names for these kind of things that are shared between the analyses. That's why I assigned you to these three analyses, so that you can keep that in mind.

Reply:
1. thx;
2. good idea. I think I will call them all "Model Parameters".